app/src/main/java/com/appunite/loudius/ui/reviewers/ReviewersScreen.kt

```diff
@@ -151,20 +168,25 @@ private fun ReviewersScreenContent(
 
 @Composable
 private fun ReviewersList(
-    reviewers: List<Reviewer>,
+    data: Data,
+    pullRefreshState: PullRefreshState,
     modifier: Modifier,
     onNotifyClick: (ReviewersAction) -> Unit,
+    isLoading: Boolean,
 ) {
-    LazyColumn(
-        modifier = modifier.fillMaxWidth(),
-    ) {
-        itemsIndexed(reviewers) { index, reviewer ->
-            ReviewerItem(
-                reviewer = reviewer,
-                index = index,
-                onNotifyClick = onNotifyClick,
-            )
+    Box(modifier = Modifier.pullRefresh(pullRefreshState)) {
+        LazyColumn(
+            modifier = modifier.fillMaxWidth(),
```

Comment:
chore: I think the common practice is to pass `modifer` parameter from a function, should be passed to the first component (in this case to Box compoment). Not to the inner component (in this case LazyColumn component).